### DISCUSSION TITLE
Endless NFT Pool connection

### DISCUSSION THREAD

**Discussion by STEELor47** [Chia-Network/chia-blockchain#8526] [Support]
Good afternoon, help solve the problem. I started connecting to the pool while the wallet was not fully synchronized. Now I cannot stop, delete or cancel the creation of the connection. What can I do in this case to fix this error ??

![image](https://user-images.githubusercontent.com/52584407/133884736-74b8fa29-5de5-44d7-bc03-b902d53821db.png)
How to stop it?

**Comment by STEELor47** [comment]
Somebody?

**Comment by loppefaaret** [comment]
could you try and check your `config.yaml` file, under the 
```
pool:
....
  pool_list:
```
section, and tell me if there is anything listed there ? and if so, is this the only plotNFT you have ?

**Comment by STEELor47** [comment]
Yes, i have. 
Now i connected on another NFT pool.
![image](https://user-images.githubusercontent.com/52584407/134241300-2ca46a61-cd1f-4dc3-a91c-f28401bd7dd5.png)
Can i just delete config.yaml file and sinch again from blockchain?

**Reply by loppefaaret** [reply]
you can delete it, but you will have to remove the wallet database atleast as well, and resync from 0 - which might be what you need to do

**Comment by STEELor47** [comment]
I'll try to reinstall chia and delete .chia.

**Comment by STEELor47** [comment]
Oh god... it doesnt help(

**Comment by BrandtH22** [comment]
@STEELor47 , the newest version of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket
